### PR TITLE
fix issue with html images

### LINKED ISSUE #4: try setting relative paths to the images, for some reason isn't working after moving into production

### PR DESCRIPTION
**Pull request by aledj2**
fix #4

---
This change is [Reviewable](https://reviewable.io/reviews/moka-guys/trend_analysis/5)

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 input_folder = "/var/www/html/mokaguys/multiqc/trend_analysis/multiqc_data"
 output_folder = "/var/www/html/mokaguys/multiqc/trend_analysis"
 images_folder = "/var/www/html/mokaguys/multiqc/trend_analysis/images/"
-logopath = images_folder + "viapathlogo.png"
+logopath = "images/viapathlogo.png"
```
Comment (natashapinto) [inline config.py:10]: Is the path to the viapathlogo correct.
If images_folder variable, please could it be removed?

File: read_qc_files.py [hunk 2]
```diff
@@ -348,7 +348,7 @@ def check_for_update():
     else:
         #print "index not modified since script run last"
```
Comment (natashapinto) [inline read_qc_files.py:349]: Can the commented lines be deleted?

**Review by aledj2 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/moka-guys/trend_analysis/5)* status: 0 of 3 files reviewed, 2 unresolved discussions (waiting on @natashapinto)

___
*[config.py, line 10 at r1](https://reviewable.io/reviews/moka-guys/trend_analysis/5#-MGc0xlENYy2K_kNRZHC-r1-10:-MGc6b_K0b2xIe-ufwt-:bclwqsy) ([raw file](https://github.com/moka-guys/trend_analysis/blob/ed04c5d2f608558e162aa99c04b6cc9ada35511e/config.py#L10)):*
<details><summary><i>Previously, natashapinto wrote…</i></summary><blockquote>

Is the path to the viapathlogo correct.
If images_folder variable, please could it be removed?
</blockquote></details>

The variable is still needed as images are saved there.

The logo path has been changed to be a relative path, inline with other images - providing the full path doesn't work as full file paths in html points to the local machine viewing/displaying the code, and not the server hosting the code (apparently)
___
*[read_qc_files.py, line 349 at r1](https://reviewable.io/reviews/moka-guys/trend_analysis/5#-MGc0xlENYy2K_kNRZHD-r1-349:-MGc725w2esJ3fTxN0Y9:bb40dcn) ([raw file](https://github.com/moka-guys/trend_analysis/blob/ed04c5d2f608558e162aa99c04b6cc9ada35511e/read_qc_files.py#L349)):*
<details><summary><i>Previously, natashapinto wrote…</i></summary><blockquote>

Can the commented lines be deleted?
</blockquote></details>

it probably could, but I think it's useful just to make it crystal clear when "debug mode" is on (when return True is uncommented below)

It's basically needed to test the code, as it overrides a test which checks if there are any new runs and if the trend analysis needs to be recalculated

**Review by natashapinto — APPROVED** [review]
> Reviewed 3 of 3 files at r1.
> *[Reviewable](https://reviewable.io/reviews/moka-guys/trend_analysis/5)* status: all files reviewed, 1 unresolved discussion (waiting on @aledj2)

___
*[read_qc_files.py, line 349 at r1](https://reviewable.io/reviews/moka-guys/trend_analysis/5#-MGc0xlENYy2K_kNRZHD-r1-349:-MGcQFGxEu8Ebm4ChXQD:b-99h6fo) ([raw file](https://github.com/moka-guys/trend_analysis/blob/ed04c5d2f608558e162aa99c04b6cc9ada35511e/read_qc_files.py#L349)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

it probably could, but I think it's useful just to make it crystal clear when "debug mode" is on (when return True is uncommented below)

It's basically needed to test the code, as it overrides a test which checks if there are any new runs and if the trend analysis needs to be recalculated
</blockquote></details>

Okay makes sense

**Review by natashapinto — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/moka-guys/trend_analysis/5)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**